weave-js/src/components/Sidebar/OutlineItemPopupMenu.tsx

```diff
@@ -161,14 +161,15 @@ const OutlineItemPopupMenuComp: React.FC<OutlineItemPopupMenuProps> = ({
   );
   const menuItems = useMemo(() => {
     const items = [];
-    if (localConfig?.id === 'Group') {
-      items.push({
-        key: 'unnest',
-        content: 'Replace with first child',
-        icon: <IconRetry />,
-        onClick: () => handleUnnest(path),
-      });
-    }
+
+    // if (localConfig?.id === 'Group') {
+    //   items.push({
+    //     key: 'unnest',
+    //     content: 'Replace with first child',
+    //     icon: <IconRetry />,
+    //     onClick: () => handleUnnest(path),
+    //   });
+    // }
```

Comment:
chatted with @tssweeney and we decided it should be ok to just hide this feature for now